hydra_python_core/doc_writer.py

```diff
@@ -629,6 +632,46 @@ def generate(self) -> Dict[str, Any]:
         return error
 
 
+class HydraLink():
+    """Template for a link property."""
+
+    def __init__(
+            self, id_: str, title: str = "",
+            desc: str = "", domain: str = "", range_: str = "") -> None:
+        """Initialize the Hydra_Link."""
+        self.id_ = id_ if "http" in id_ else "vocab:{}".format(id_)
+        self.range = range_
+        self.title = title
+        self.desc = desc
+        self.domain = domain
+        self.supportedOperation = list()  # type: List
+
+    def add_supported_op(
```

Comment:
Again, we don't need these operations here because there is no endpoint for hydra:Link. It is a class defined in the spec, not in the API Doc.